### PR TITLE
Administration: add admin link in the profile menu

### PR DESCRIPTION
**Pull request by Pineirin**
* closes: #66

Admin:
Screenshot from 2022-09-16 19-47-54
Screenshot from 2022-09-16 19-48-45
Screenshot from 2022-09-19 10-49-06

Not admin:
Screenshot from 2022-09-19 10-51-59
Screenshot from 2022-09-19 10-52-12

### CODE REVIEW

File: invenio_administration/menu/menu.py [hunk 1]
```diff
@@ -62,6 +65,20 @@ def register_menu_entries(self, flask_menu_instance, menu_key="admin_navigation"
                     icon=icon,
                 )
 
+    def register_admin_entry(self, current_menu, endpoint):
+        """Register all menu items to a flask menu instance."""
```
Comment (jrcastro2) [inline invenio_administration/menu/menu.py:69]:
```suggestion
        """Register administration entry as the last one."""
```

File: invenio_administration/menu/menu.py [hunk 2]
```diff
@@ -62,6 +65,20 @@ def register_menu_entries(self, flask_menu_instance, menu_key="admin_navigation"
...
+        current_menu.submenu("settings.administration").register(
+            f"{endpoint}.dashboard",
+            _(
+                "%(icon)s Administration",
+                icon=make_lazy_string(
+                    lambda: f'<i class="{current_theme_icons.cogs}"></i>'
+                ),
+            ),
+            order=6,
+            visible_when=lambda: current_user.roles.count("admin") > 0,
```
Comment (jrcastro2) [inline invenio_administration/menu/menu.py:79]: I would increase the order as we might add new items and we always want it to be the last one.

```suggestion
            order=100,
            visible_when=lambda: current_user.has_role("admin"),
```

File: invenio_administration/menu/menu.py [hunk 3]
```diff
@@ -62,6 +65,20 @@ def register_menu_entries(self, flask_menu_instance, menu_key="admin_navigation"
...
+        current_menu.submenu("login.administration").register(
+            f"{endpoint}.dashboard",
+            _(
+                "%(icon)s Administration",
+                icon=make_lazy_string(
+                    lambda: f'<i class="{current_theme_icons.cogs}"></i>'
+                ),
+            ),
+            order=1,
+            visible_when=lambda: current_user.roles.count("admin") > 0,
```
Comment (kpsherva) [inline invenio_administration/menu/menu.py:79]:
```suggestion
            visible_when=lambda: current_user.has_role("admin") > 0,
```

File: invenio_administration/menu/menu.py [hunk 4]
```diff
@@ -62,6 +65,20 @@ def register_menu_entries(self, flask_menu_instance, menu_key="admin_navigation"
                     icon=icon,
                 )
 
+    def register_admin_entry(self, current_menu, endpoint):
+        """Register administration entry as the last one."""
+        current_menu.submenu("login.administration").register(
```
Comment (jrcastro2) [inline invenio_administration/menu/menu.py:70]: Question: IMHO not sure `login` is the most intuitive name, maybe `navbar-menu` would be more explicit? WDYT?

**Review by kpsherva — APPROVED** [review]

**Pull request merged**